utils/docker.go

```diff
@@ -310,3 +312,26 @@ func GetPFEPort() string {
 	}
 	return ""
 }
+
+// IsTCPPortAvailable checks to see if a specified port is available
```

Comment:
should the comment indicate that the function will pick the next available port ?

Reply:
You are correct, yes. It doesn't only find a port, it also returns it.